### PR TITLE
fix error for checkpoint when space doesn't exist in storageNode.

### PR DESCRIPTION
**Pull request by bright-starry-sky**

### CODE REVIEW

File: src/meta/processors/admin/SnapShot.cpp [hunk 1]
```diff
@@ -88,19 +88,38 @@ cpp2::ErrorCode Snapshot::blockingWrites(storage::cpp2::EngineSignType sign) {
...
 }
 
+std::set<HostAddr> Snapshot::getHostsBySpace(GraphSpaceID space) {
+    folly::SharedMutex::ReadHolder rHolder(LockUtils::spaceLock());
+    std::set<HostAddr> hosts;
+    auto prefix = MetaServiceUtils::partPrefix(space);
+    std::unique_ptr<kvstore::KVIterator> iter;
+    auto kvRet = kv_->prefix(kDefaultSpaceId, kDefaultPartId, prefix, &iter);
+    if (kvRet != kvstore::ResultCode::SUCCEEDED) {
+        return {};
```
Comment (darionyaphet) [inline src/meta/processors/admin/SnapShot.cpp:111]: maybe it is an error ?

Comment (bright-starry-sky) [reply src/meta/processors/admin/SnapShot.cpp:111]: em... What's the difference between '{}' and 'empty list'?

Comment (bright-starry-sky) [reply src/meta/processors/admin/SnapShot.cpp:111]: > maybe it is an error ?

I see what you mean. let me fix it.

**Comment by bright-starry-sky** [comment]
Please hold this PR.

**Comment by dangleptr** [comment]
> Please hold this PR.

Any concerns?

**Comment by bright-starry-sky** [comment]
> > Please hold this PR.
> 
> Any concerns?

By @darionyaphet's comment . An invalid checkpoint will be created when kv_->prefix error occur.

**Comment by bright-starry-sky** [comment]
Addressed darionyaphet's comment. 
Ready to review again. Thanks.

**Pull request merged**